### PR TITLE
Enhanced :removed the start button under  worklist tab

### PR DESCRIPTION
**Pull request by FelixKiprotich350**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary
removed the start button under worklist tab which was changing the status of a test to in progress (duplicate of what pickup request does)

## Screenshots
[radiology-startbutton-removed.webm](https://github.com/user-attachments/assets/72e4da13-0071-4c2a-b899-21fce6da2982)

### CODE REVIEW

**Comment by FelixKiprotich350** [comment]
@ojwanganto check this

**Comment by ojwanganto** [comment]
Thanks, @FelixKiprotich350. CC @donaldkibet

File: src/radiology-tabs/test-ordered/tests-ordered.component.tsx [hunk 1]
```diff
@@ -10,24 +10,22 @@ export const TestsOrdered: React.FC = () => {
...
+        showStatus={true}
+        showStartButton={false}
+        showActions={true}
+        showOrderType={false}
+        actions={[
+          {
+            actionName: "add-radiology-to-worklist-dialog",
+            displayPosition: 1,
+          },
+          { actionName: "reject-radiology-order-dialog", displayPosition: 2 },
```
Comment (donaldkibet) [inline src/radiology-tabs/test-ordered/tests-ordered.component.tsx:26]:
```suggestion
          { actionName: "reject-radiology-order-dialog", order: 2 },
```

**Review by donaldkibet — APPROVED** [review]
LGTM Feel free to merge once you have fix it

File: src/radiology-tabs/test-ordered/tests-ordered.component.tsx (hunk 1 above)
Comment (FelixKiprotich350) [reply src/radiology-tabs/test-ordered/tests-ordered.component.tsx]: fixed

**Pull request merged**